pages/bluedialog/bluedialog.page.tsx

```diff
@@ -0,0 +1,98 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  DatePicker,
+  ExpandableSection,
+  Form,
+  FormField,
+  Header,
+  Input,
+  Select,
+  SpaceBetween,
+  Textarea,
+} from '~components';
+
+import styles from './styles.scss';
+
+export default function Bluedialogbox() {
+  const [showDialog, setShowDialog] = React.useState(true);
+  // Inputs
+  const [dateValue, setDateValue] = React.useState('');
+  const [amountValue, setAmountValue] = React.useState('0');
+  const [descriptionValue, setDescriptionValue] = React.useState('');
+  const [selectedService, setSelectedService] = React.useState({ label: 'Cloudwatch', value: 'Cloudwatch' });
+
+  function submitData() {
+    console.log({ dateValue, amountValue, descriptionValue, selectedService });
+  }
+  function skipDialog() {
+    setShowDialog(false);
+  }
+
+  return showDialog ? (
```

Comment:
Let's not remove the dialog at all, we can console.log instead. I think overall the example here could be simpler.

Reply:
Done in latest revision.